### PR TITLE
Fix channel mentions #57

### PR DESCRIPTION
**Pull request by nzlosh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,3 +90,15 @@ Ensure the bot is also subscrbed to the following events:
...
+
+### Bot Admins
+Slack changed the way users are uniquely identified from display name `@some_name` to user id `Uxxxxxx`.
+Errbot configuration will need to be updated before administrators can be correctly identified aginst
+the ACL sets.
+
+The UserID is in plain text format.  It can be found in the the Slack full profile page or using the `!whoami` command (`person` field).
+
+Because BOT_ADMINS is defined as plain text User IDs, they can not be used to send notifications.  The mention format
+`<@Uxxxxx>` must be used in the BOT_ADMINS_NOTIFICATIONS configuration setting for errbot to initiate message to bot administrators.
```
Comment (alpinweis) [inline README.md:104]: In my testing this format dint work:
`BOT_ADMINS = ("<@Uxxxxx>," "<@Uyyyyy>")`

But this one worked:
`BOT_ADMINS = ("Uxxxxx",  "Uyyyyy")`

**Comment by alpinweis** [comment]
nice to see the `channelid` property added to `StackRoom`.
Until now, one had to do something like this to get it w/o throwing an error.
```
channel_id = msg.to.id if msg.is_group else msg.to.channelid
```
Would this also be added to other backends (e.g. for compatibility and consistency)?

File: README.md (hunk 1 above)
Comment (nzlosh) [reply README.md:104]: Yes, that looks correct.  `BOT_ADMINS` takes plain text user ids

**Comment by nzlosh** [comment]
The original `slack` backend `SlackRoom` class has `channelid` which is an alias to the `id` property.  There's no intention to generalise `channelid` across backends at this point.  If it makes sense to do that, PRs would be welcome.

**Pull request merged**